app/redhat_insights/default/data/ui/views/main_dashboard.xml

```diff
@@ -639,20 +919,78 @@
+				},
+				"query": "index=redhatinsights AND  application=vulnerability\n  | spath \n  | spath path=events{} output=events\n  | stats by _time, events, application, event_type, account_id, context.display_name\n  | mvexpand events \n  | eval _raw=events\n  | kv\n  | eval  has_rule='payload. has_rule'\n  | table _time,  has_rule\n  | where has_rule=True\n  | timechart span=24h aligntime=latest count( has_rule) as count | appendpipe [| stats count | where count=0 | addinfo | eval time=info_min_time.\" \".info_max_time | makemv time | mvexpand time | table time count | rename time as _time ]"
+			},
+			"name": "search_cve_with_security_rules_24hrs"
+		},
+		"ds_ZlqEH4oo_ds_HRRsnn9w": {
+			"type": "ds.search",
+			"options": {
+				"query": "index=redhatinsights AND application=vulnerability\n  | spath \n  | spath path=events{} output=events\n  | stats by _time, events, application, event_type, account_id, context.display_name\n  | mvexpand events \n  | eval _raw=events\n  | kv\n  | eval  has_rule='payload. has_rule'\n  | table _time,  has_rule\n  | where now()>relative_time(_time, \"+30d\")\n  | where has_rule=True\n  | timechart span=24h aligntime=latest count(has_rule) as count \n  | appendpipe [| stats count | where count=0 | addinfo | eval time=info_min_time.\" \".info_max_time | makemv time | mvexpand time | table time count | rename time as _time ]"
+			},
+			"name": "search_cve_with_security_rules_30days"
+		},
+		"ds_Th8SaLkj_ds_cflXieRi_ds_snKvID23": {
+			"type": "ds.search",
+			"options": {
+				"queryParameters": {
+					"earliest": "-48h",
+					"latest": "now"
+				},
+				"query": "index=redhatinsights AND event_type=any-cve-known-exploit\n  | spath \n  | spath path=events{} output=events\n  | stats by _time, events, application, event_type, account_id, context.display_name\n  | mvexpand events \n  | eval _raw=events\n  | kv\n  | eval known_exploit='payload.known_exploit'\n  | table _time, known_exploit\n  | where known_exploit=True\n  | timechart span=24h aligntime=latest count(known_exploit) as count | appendpipe [| stats count | where count=0 | addinfo | eval time=info_min_time.\" \".info_max_time | makemv time | mvexpand time | table time count | rename time as _time ]"
+			},
+			"name": "search_cve_with_know_exploits_24hrs"
+		},
+		"ds_U2K2gqcn_ds_8HzUA9oC_ds_HRRsnn9w": {
+			"type": "ds.search",
+			"options": {
+				"query": "index=redhatinsights AND event_type=any-cve-known-exploit\n  | spath \n  | spath path=events{} output=events\n  | stats by _time, events, application, event_type, account_id, context.display_name\n  | mvexpand events \n  | eval _raw=events\n  | kv\n  | eval known_exploit='payload.known_exploit'\n  | table _time, known_exploit\n  | where known_exploit=True\n  | timechart span=24h aligntime=latest count(known_exploit) as count | appendpipe [| stats count | where count=0 | addinfo | eval time=info_min_time.\" \".info_max_time | makemv time | mvexpand time | table time count | rename time as _time ]"
```

Comment:
This query is lacking queryParameters.earliest to narrow the search to 30 days for performance.

Reply:
Fixed here: 4544bec57ec8b28f617bc76d258ab80090bcf3ba